django/src/rdwatch/models/site_observation.py

```diff
@@ -14,13 +14,11 @@ class SiteObservation(models.Model):
     siteeval = models.ForeignKey(
         to='SiteEvaluation',
         on_delete=models.CASCADE,
-        help_text='The site evaluation associated with this observation.',
         db_index=True,
     )
     label = models.ForeignKey(
         to='ObservationLabel',
         on_delete=models.PROTECT,
-        help_text='Site observation classification label',
```

Comment:
I don't think they're particularly helpful descriptions, but any particular reason to remove these?

Reply:
CI for linting was fighting with pre-commit and I think flake8 or black were arguing with line length and causing new migrations so I just deleted the text.